products/build.gradle.kts

```diff
@@ -12,13 +13,37 @@ repositories {
 }
 
 dependencies {
-    implementation("org.springframework.boot:spring-boot-starter-jersey")
-    implementation ("org.springframework.boot:spring-boot-starter-webflux")
+    implementation("org.springframework.boot:spring-boot-starter-webflux")
     developmentOnly("org.springframework.boot:spring-boot-devtools")
     testImplementation("org.springframework.boot:spring-boot-starter-test")
     testImplementation("io.projectreactor:reactor-test")
 }
-tasks.compileJava{
+
+application {
+    mainClass.set("de.baleipzig.products.ProductsApplication")
+}
+
```

Comment:
hier wird der Container erstellt, das java Programm in den Container gepackt und danach der Container im cluster registriert oder ?

Reply:
Nein application ist nur für das Starten ohne Docker/Kubernetes